### ISSUE TITLE
Error creating example cluster with aws_attributes

### ISSUE THREAD

**Issue by tjheslin1** [innovationnorway/terraform-provider-databricks#40]
Hi there,

Going by `interface {} is int, not int32` I tried converting the `aws_attributes` to strings which didn't help. Would love to be able to use the project going forward!

### Terraform Version
```
Terraform v0.12.24
+ provider.databricks (unversioned)
```

### Affected Resource(s)
- databricks_cluster

### Terraform Configuration Files
```hcl
provider "databricks" {
  host  = "https://xxxxxxx.cloud.databricks.com"
  token = "xxxxxxx"

resource "databricks_cluster" "example" {
  cluster_name  = "example"
  spark_version = "6.3.x-scala2.11"
  node_type_id  = "m4.large"

  autoscale {
    min_workers = 2
    max_workers = 8
  }

  aws_attributes {
    first_on_demand = 1
    ebs_volume_type = "GENERAL_PURPOSE_SSD"
    ebs_volume_count = 3
    ebs_volume_size = 100
  }

  autotermination_minutes = 10
}
```

### Panic Output
https://gist.github.com/tjheslin1/571d4449194b7b0de296f2b3f287111b

### Expected Behavior
Cluster to have been created

### Actual Behavior
panic

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### Important Factoids
N/A

**Comment by joakimhellum** [comment]
@tjheslin1 Very sorry about this.

To be honest we only recently started working on this provider and we didn't set up accounts for running acceptance tests in AWS yet. However we do plan to support all configurations and add the tests for them.

I will start now on fixing the issues and update tests for the `databricks_cluster` to include AWS attributes.

Hopefully this is merged into master within next 24 hours :) Thanks for your patience

**Comment by tjheslin1** [comment]
Hey! Thanks for the quick response. Going forward I should be able to contribute. My golang isn't great but if I do contribute I'll follow the style that's already there. Currently getting to grips with the Databricks API.

**Comment by joakimhellum** [comment]
@tjheslin1 Thanks! We highly appreciate any contribution (issues or PRs). 
Made attempt to fix this in #42 
Note: Will add more test cases during weekend and then release `v0.1.0-alpha.2` version
